translator/translate/otel/common/common.go

```diff
@@ -174,6 +177,17 @@ func (e *MissingKeyError) Error() string {
 	return fmt.Sprintf("%q missing key in JSON: %q", e.ID, e.JsonKey)
 }
 
+// A DebugLoggingEnabledError occurs when app signals monitoring is not enabled for
+// an app signals translator.
+type DebugLoggingEnabledError struct {
```

Comment:
This is very specific to the debug exporter. Move it out of `common`. I don't think you need a custom error for this either.